### DISCUSSION TITLE
Is there a syntax to querying the library version of mermaid?

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [mermaid-js/mermaid#5537] [Q&A] [answered]
Mermaid is so awesome, and so many projects have it embedded, but because its developed here unless someone on the embedding project takes great care of it, it is not frequently updated. This makes sense, especially when following the [II Dependencies of 12 Factor Apps](https://en.wikipedia.org/wiki/Twelve-Factor_App_methodology#The_Twelve_Factors).

Since mermaid is a renderng engine, it makes sense to me that it should be able to let us know which version of the library (maybe even the commit hash from the github project) is being used. This would make troubleshooting, syntax refernce or just requesting an upgrade from the maintainer of the embedding project much easier.

Is it possible today? I couldn't find any mention of such an option.
If not, I doubt I have the know how on how to do it, but I'll be happy to open a feature request.

**Comment by oeyoews** [comment, marked as answer]
```mermaid
info
```

![image](https://github.com/user-attachments/assets/fb0d53ee-3037-445b-967c-484b333b7665)

**Reply by Lockszmith-GH** [reply]
Amazing! so simple.

Latest release of mermaid is [v11.0.2](https://github.com/mermaid-js/mermaid/releases/tag/mermaid%4011.0.2), with v11, released a day or so ago.
GitHub's rendering right now shows [v10.9.1](https://github.com/mermaid-js/mermaid/releases/tag/v10.9.1) from May. 

Searching the repo armed with this new knowledge, I see it used to be `info showInfo` [back in 2015](https://github.com/mermaid-js/mermaid/issues/111#issuecomment-71219088) and then the `showInfo` was dropped sometime around Aug 2023.

This should be part of the documentation. Hopefully this Q&A would help others like me.

Thanks!

P.S Good to know that GitHub are keeping the engine up to date, now to test it where it was failing for me 🤓
